### ISSUE TITLE
CSGO Set player kills/deaths

### ISSUE THREAD

**Issue by high-voltage** [Source-Python-Dev-Team/Source.Python#91]
I tried resetting Player.kills & deaths to 0. It actually works, but as soon as the next round starts, the game sets them back to the old values.

```
from players.entity import Player
from players.helpers import index_from_userid
from events import Event

@Event('player_say')
def player_say(event):
    player = Player(index_from_userid(event.get_int('userid')))
    player.kills = 0
    player.deaths = 0
```

**Comment by high-voltage** [comment]
Found out that this bug only occurs if you reset kills/deaths when the round ended. Otherwise it works fine. So a small workaround for plugin makers would be using a dict/list which adds the player to it that wants to reset his score, and then just reset it on round start. I don't think it's worth finding a fix for it, because it's a really weird bug, so I guess this can be closed.

**Comment by Ayuto** [comment]
Thanks for the update! I think it would be nice if setting kills/deaths is reliable at every phase of the round and in every game.

I just did a few more tests and found out that using ``player.increment_frag_count(-player.kills, 0)`` works fine. Internally, it's setting another attribute, which is probably synchronized at some point with the attribute we set. Note that the ``0`` is actually completely redundant, but the engine requires us to pass a seconds integer, although it's never used in that method...

**Comment by high-voltage** [comment]
Tested now as well, ```player.increment_frag_count(-player.kills, 0)``` and ```player.increment_death_count(-player.deaths)``` works fine.

I think ```player.kills``` and ```player.deaths``` should now be only get attributes, and for setting, the api should use the other functions. Atleast for csgo, I haven't tested this with other games.

**Comment by satoon101** [comment]
I am working on this now.  Does this affect assists, as well?  We do already have increment_assists_count for CS:GO in our data.

**Comment by satoon101** [comment]
Never mind.  I did some testing and found out it does affect assists, too.  I will commit the changes shortly.

**Comment by satoon101** [comment]
I forgot to reference and close this in the commit message, but this is fixed in 5edaf53dd7539cf2852206d88839ac544b214513.

Thank you for reporting this bug!
